samples/matter/lock/README.rst

```diff
@@ -460,6 +481,56 @@ Upgrading the device firmware
 
 To upgrade the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial of the Matter documentation.
 
+.. _matter_lock_sample_remote_access_with_pin:
+
+Testing remote access with PIN code credential
+==============================================
+
+.. note::
+   You can test the PIN code credential support with any Matter compatible controller.
+   The following steps use the CHIP Tool controller as an example.
+
+After building the sample and programming it to your development kit, complete the following steps to test remote access with PIN code credential:
+
+1. |connect_kit|
+#. |connect_terminal_ANSI|
+#. Wait until the device boots.
+#. Commission an accessory to the Matter network by following the steps described in the `Commissioning the device`_ section.
+#. Make the door lock require a PIN code for remote operations:
+
+   .. code-block:: console
+
+      ./chip-tool doorlock write require-pinfor-remote-operation 1 1 1 --timedInteractionTimeoutMs 5000
+
+#. Add the example ``Home`` door lock user with the following CHIP Tool command:
+
+   .. code-block:: console
+
+      ./chip-tool doorlock set-user 0 1 Home 0 1 0 0 1 1 --timedInteractionTimeoutMs 5000
```

Comment:
I know that user may read the command description, but arguments like '0 1 0 1 0 0 1 1' look a little bit confusing :)

Reply:
Ok, I will add some explanation with cooperation with @wiba-nordic .